src/quic/node_quic_socket.cc

```diff
@@ -269,11 +270,17 @@ QuicSocket::QuicSocket(
 
   EntropySource(token_secret_, kTokenSecretLen);
 
+  wrap->DefineOwnProperty(
+      env()->context(),
+      env()->block_list_string(),
+      block_list_->object(),
+      PropertyAttribute::ReadOnly).FromJust();
+
   wrap->DefineOwnProperty(
       env()->context(),
       env()->state_string(),
       state_.GetArrayBuffer(),
-      PropertyAttribute::ReadOnly).Check();
+      PropertyAttribute::ReadOnly).FromJust();
```

Comment:
I think `.FromJust()` gives an “unused result” warning, which is why we used `.Check()`.

(Of course, the better way is to use neither :))

Reply:
heh... I'll switch it back to Check which is most common but I really don't like this part of the v8 api lol